eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  - name: DailyBranchVariableName
```

Comment:
Do we gain much by letting this variable name be set? If not I'd just give it the same value everywhere and folks can then use that variable if they need. That is what we have done in other places like DefaultBranch etc,

Reply:
I set it to a different value in different places. In general when setting a global variable I want to give the user the opportunity to specify a variable name. 

This came up in other places where I was working with two repos and setting the name of the default branch. -- https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/pipelines/templates/steps/set-default-branch.yml

In such a case you have remember which repo was most recently used to set `$(DefaultBranch)`.